Okta/OktaOidc/OktaOidc.swift

```diff
@@ -14,6 +14,7 @@ public class OktaOidc: NSObject {
 
     // Cache Okta.plist for reference
     @objc public let configuration: OktaOidcConfig
+    @objc public weak var delegate: OktaOidcHTTPProtocol?
```

Comment:
```suggestion
    @objc public weak var httpNotifyDelegate: OktaOidcHTTPProtocol?
```

Reply:
I think we decided to make this part of OktaOidcConfig. The reason why it should be part `OktaOidcConfig` class is that you can also propagate `httpNotifyDelegate` to `OktaOidcStateManager`

Reply:
Yes, but `OktaOidcConfig ` conforms to `OktaOidcConfig ` protocol and I can't make this new protocol conforms to Codable protocol and available in ObjC. That's why I moved it to `OktaOidc` class.

Reply:
Ah, I see. Please remove `Codable` compliance for `OktaOidcConfig`, it is not required as `OktaOidcStateManager` doesn't store reference to `OktaOidcConfig` in latest version. Please rework your code to use `OktaOidcConfig`